.gitignore

```diff
@@ -133,4 +133,9 @@ $RECYCLE.BIN/
 *.lnk
 
 
-# End of https://www.gitignore.io/api/osx,node,windows
\ No newline at end of file
+# End of https://www.gitignore.io/api/osx,node,windows
+
+# Ignore root level dockerfiles
+/Dockerfile
+/docker-compose.yml
+/.dockerignore
```

Comment:
@alex-gagnon Is there anything in these files that folks will have to have locally in order to get things to work?

Reply:
They will need Docker installed. On Windows, they'll have to have Windows 10 Pro. I haven't used this on linux, so not sure of the requirements there.

Reply:
@alex-gagnon Can you add instructions on how to install Docker?

Reply:
Yes, I am updating the README file now